packages/javascript/src/version.ts

```diff
@@ -0,0 +1 @@
+export const VERSION = "1.0.0-beta.1"
```

Comment:
Now that this file is auto generated, do we maybe want to add a comment like:
```
# Do not touch this file, auto-generated by scripts/create-version
```

Reply:
Good idea! Fixed in 11d96e7